source/utils/versions.ts

```diff
@@ -0,0 +1,6 @@
+export function ensure_correct_node_version() {
+    const version_arr = process.versions.node.split('.').map(Number)
+    if ((version_arr[0] >= 15 && version_arr[1] >= 6) || version_arr[0] < 12) {
```

Comment:
Would be great to add a link to the github issue as a comment to explain why we do that

Reply:
Good idea, will do